Examples/Swift/AnnotationViewMultipleExample.h

```diff
@@ -0,0 +1,13 @@
+//
+//  AnnotationViewMultipleExample.h
+//  Examples
+//
+//  Created by Nadia Barbosa on 12/13/16.
+//  Copyright © 2016 Mapbox. All rights reserved.
+//
+
```

Comment:
it was missing from my version, not sure if it's needed. Can remove it if it's not necessary :)